doc/Command-Reference.md

```diff
@@ -3997,12 +4026,17 @@ This command displays a list of NTP peers known to the server as well as a summa
 - Example:
   ```
   admin@sonic:~$ show ntp
+  synchronised to NTP server (204.2.134.164) at stratum 3
```

Comment:
Is this change intended to be part of this?

Reply:
Not sure, it must come from a git pull. Should I remove this or leave there?